.travis.yml

```diff
@@ -46,7 +53,7 @@ script:
     esac
 
 after_success:
-  - if \[ "$TASK" == "BUILD" \] && \[ "$TRAVIS_JDK_VERSION" == "oraclejdk8" \]; then
+  - if \[ "$TASK" == "BUILD" \] && \[ "$TRAVIS_JDK_VERSION" == "oraclejdk8" \] && \[ "$TRAVIS_OS_NAME" = linux \]; then
```

Comment:
Suggestion: You could add another variable to control which job handles the coverage report.